packages/lab/src/portal/Portal.tsx

```diff
@@ -67,25 +53,6 @@ export const Portal = forwardRef<HTMLElement, PortalProps>(function Portal(
 
   const container = getContainer(containerProp) ?? document.body;
 
-  useIsomorphicLayoutEffect(() => {
```

Comment:
How will we approach the Desktop use-case with Portal ?

Reply:
What are our thoughts on controlling the placement of content?
I see floating-window has middleware, I guess we create a way to use this?

Most folks get a default but there is a way that it can be overridden for the Desktop use-case etc ?
Can you point us at any code which might exist already?

Reply:
Still trying to figure it out